presto-docs/src/main/sphinx/release/release-0.248.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.248
+=============
+
+**Highlights**
+==============
+* New aggregation function MAP_UNION_SUM.
+* New scalar function DESTRUCTURE_TDIGEST.
+* Support for session property managers, that can override session properties set by a user.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve output buffer memory tracking to reduce lock contention.
+* Add support for overriding session properties using session property managers :doc:`/admin/session-property-managers`. Setting ``overrideSessionProperties`` to true will cause the property to be overridden and remain overridden even if subsequent rules match the property but don't have ``overrideSessionProperties`` set.
+* Add support to drop multiple UDFs at the same time.
+* Add a REST endpoint ``/v1/taskInfo/{{taskId}}`` on the coordinator to get TaskInfo without needing to go directly to the worker's endpoint.
+* Add new aggregation function :func:`map_union_sum`.
+* Add new scalar function DESTRUCTURE_TDIGEST to expose the internal TDigest data structure as a ROW.
```

Comment:
```
Add new scalar function :func:`destructure_tdigest` to expose the internal ``TDigest`` data structure as a ``ROW``
```

Reply:
There's no documentation for this yet, so we can't add the function link.

Reply:
destructure_tdigest doesn't have any documentation, but I've surrounded TDigest and ROW with the double back quotes.

Reply:
As per a conversation with @rschlussel - we should not include any function mention in the release notes if there is no documentation. lets remove the mention of this function from here and the highlights as well

Reply:
Ok, I've removed references to destructure_tdigest both in details and highlights section